bot/seasons/evergreen/bookmark.py

```diff
@@ -0,0 +1,78 @@
+import logging
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Bookmark(commands.Cog):
+    """A cog for Bookmarking a message."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="bookmark", aliases=("bm", "pin"))
+    async def bookmark(self, ctx: commands.Context, jump_url: str = None, *args) -> None:
+        """Bookmarks a message."""
+        if jump_url is not None and jump_url != "*":
+            if "discordapp.com/channels" not in jump_url:
+                await ctx.send("INVALID URL")
+                await ctx.send("An example of the command can be \n"
+                               "!bm https://discordapp.com/channels"
+                               "/267624335836053506/267631170882240512/"
+                               "554639398130548746"
+                               " Some of the hints here")
+                return
+        if jump_url is None or jump_url == "*":
+            channel = ctx.channel
+            async for x in channel.history(limit=2):
+                message_id = x.id
+            try:
+                message = await channel.fetch_message(message_id)
+            except discord.NotFound:

```

Comment:
I don't think it can ever trigger, right?

Reply:
It can if the channel is empty , just a safety still .

Reply:
Actually, it doesn't trigger if there is no message in the channel, but that's not a problem